skiko/build.gradle.kts

```diff
@@ -1168,7 +1168,8 @@ fun skikoJvmRuntimeJarTask(
 ) = project.registerSkikoTask<Jar>("skikoJvmRuntimeJar", targetOs, targetArch) {
     dependsOn(awtJar)
     val target = targetId(targetOs, targetArch)
-    archiveBaseName.set("skiko-$target")
+    archiveBaseName.set("skiko")
```

Comment:
Looks good, but let's wait merging until we release Compose 1.5.10 (we need a hotfix for skiko)

Reply:
Sure, no problem